libs/langgraph/langgraph/pregel/__init__.py

```diff
@@ -352,86 +361,287 @@ def managed_values_dict(self) -> dict[str, ManagedValueSpec]:
             if is_managed_value(v)
         }
 
-    def get_state(self, config: RunnableConfig) -> StateSnapshot:
-        """Get the current state of the graph."""
-        if not self.checkpointer:
-            raise ValueError("No checkpointer set")
+    def _get_nodes_and_channels(
```

Comment:
in langgraph we don't add private methods to the class, sorry. if you utility methods they should be functions outside of the class